login/styles.css

```diff
@@ -0,0 +1,101 @@
+.body {
+  background: #f7f7f7;
+}
+.login-container {
+  margin: 2rem auto;
+  padding: 2.5rem;
+  width: 30%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #ffffff;
+  box-shadow: 0px 4px 50px rgba(0, 0, 0, 0.05);
+  border-radius: 10px;
+}
+.login-title {
+  display: flex;
+  justify-content: center;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 48px;
+  color: #000000;
+}
+.email-address {
+  margin-top: 2.6rem;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #282828;
+}
+.email-input {
+  padding: 0.382rem 12.531rem 0.567rem 1.084rem;
+  border-radius: 0.313rem;
+  background: #ffffff;
+  border: 1px solid #d0d0d0;
+}
+.password {
+  margin-top: 2rem;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #282828;
+}
+.password-input {
+  padding: 0.382rem 12.531rem 0.567rem 1.084rem;
+  border-radius: 0.313rem;
+  font-style: normal;
+  font-weight: normal;
+  color: #afafaf;
+  background: #ffffff;
+  border: 1px solid #d0d0d0;
+}
+.checkbox-container {
+  margin-top: 2rem;
+  display: flex;
+  flex-direction: row;
+}
+.checkbox {
+  height: 1rem;
+  width: 1rem;
+}
+.checkbox-text {
+  margin-left: 1rem;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #000000;
+}
+.checkbox-forgot-text {
+  margin-left: 2.063rem;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #3b82f6;
+}
+.login-btn {
+  margin-top: 2.06rem;
```

Comment:
it is advisable to avoid use of margin-top,bottom,right,left properties to centre elements due to responsiveness.Try to use flex:centre/grid properties instead.